src/legacy/ui/public/chrome/api/injected_vars.ts

```diff
@@ -18,11 +18,11 @@
  */
 
 import { cloneDeep } from 'lodash';
-import { InjectedMetadataStart } from '../../../../../core/public/injected_metadata';
+import { InjectedMetadata } from '../../../../../core/public/';
 
-let newPlatformInjectedVars: InjectedMetadataStart;
+let newPlatformInjectedVars: InjectedMetadata.InjectedMetadataStart;
```

Comment:
I'm fine with having domain name as a prefix for start contract, it's useful for manual search